### PR TITLE
Changes Reaction's z-index place + adds size to the user icon in Jazzbar

### PR DESCRIPTION
**Pull request by mike-lvov**
Changes Reaction's z-index place + adds size to the user icon in Jazzbar
closes https://github.com/sparkletown/internal-sparkle-issues/issues/324

### CODE REVIEW

**Review by denisdimitrov — APPROVED** [review]
LGTM!

File: src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const Reaction = styled.div<ReactionProps>`
   ${({ reactionPosition }) =>
     reactionPosition === "right" ? reactionRight : reactionLeft};
   top: -25px;
-  z-index: 1000;
+  z-index: 10;
```
Comment (0xdevalias) [inline src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts:37]: Add this to the z-access helper

Comment (mike-lvov) [reply src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts:37]: This is a styled component file

Comment (0xdevalias) [reply src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts:37]: Ah, missed that. :( The sooner we can remove those the better IMO

Comment (mike-lvov) [reply src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts:37]: > Ah, missed that. :( The sooner we can remove those the better IMO


Agreed

**Pull request merged**